pkg/framework/aries/api/vdr/vdr.go

```diff
@@ -35,6 +41,9 @@ type VDR interface {
 	Read(did string, opts ...resolve.Option) (*did.DocResolution, error)
 	Store(doc *did.Doc, by *[]doc.ModifiedBy) error
 	Build(keyManager kms.KeyManager, opts ...create.Option) (*did.DocResolution, error)
+	Update(did string, opts ...update.Option) error
+	Recover(did string, opts ...recovery.Option) error
```

Comment:
Recover isn't part of did-core. https://www.w3.org/TR/did-core/#method-operations

Reply:
@troyronda logically it is an update but it is hard to mash update and recover. Recover is not supposed to be used lightly.

Reply:
We can omit recover from interface for now and see how/if it will be used in apps